### PR TITLE
Feature/#4 decouple preprocessor from base configuration

### PR DESCRIPTION
**Pull request by leroykorterink**
- **Decoupled preprocessor from base configuration**
   There are packages for 3 different syntaxes now, use the package for the syntax that you use in your project.
  - `@mediamonks/stylelint-config-css`
  - `@mediamonks/stylelint-config-scss`
  - `@mediamonks/stylelint-config-css-in-js`
- **Added 3 new rules to the base configuration:**
  - `'csstools/use-logical'`
  - `'order/order'`
- Added packages to test stylelint configuration
- Added github workflow action to publish packages
- Added eslint, prettier and pre-commit hook

### CODE REVIEW

File: .github/workflows/bump-version-and-publish.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# NOTE: taken from https://github.com/MylesBorins/node-osc/blob/main/.github/workflows/bump-version.yml
+
+name: Bump version & Publish
+
+on:
+  workflow_dispatch:
+    inputs:
```
Comment (ThaNarie) [inline .github/workflows/bump-version-and-publish.yml:7]: You can use `choice` type now :)

See https://github.com/mediamonks/pota/blob/main/.github/workflows/bump-and-publish.yml#L6-L36 for an example

**Review by ThaNarie — APPROVED** [review]

File: .github/workflows/bump-version-and-publish.yml (hunk 1 above)
Comment (leroykorterink) [reply .github/workflows/bump-version-and-publish.yml:7]: Ah will update it! I copied this configuration from the @mediamonks/eslint-config repository :)

**Comment by ReneDrie** [comment]
I don't agree with having to order the CSS properties Alphabetical, don't see any reason as to why we would do this and we had discussions about this but never got one proper answer.
Without ordering anything I can approve this PR

**Review by ThaNarie — APPROVED** [review]

**Comment by leroykorterink** [comment]
@ReneDrie I've removed the property order rule, we'll add the [stylelint-devtool-prop-order](https://github.com/csstools/stylelint-devtool-prop-order) rule when it's published on npm.

**Review by ReneDrie — APPROVED** [review]

**Review by ThaNarie — APPROVED** [review]

**Pull request merged**